src/goose/job.clj

```diff
@@ -15,9 +15,14 @@
    :retry-opts     retry-opts
    :enqueued-at    (u/epoch-time-ms)})
 
+(defn retried? [job]
+  (if (get-in job [:state :error])
```

Comment:
```suggestion
  (boolean (get-in job [:state :error]))
```

Reply:
I have forgotten to replace this line with `job/retried?`

Reply:
@alishamohanty I meant that we can replace the if condition with a boolean type-cast. If `[:state :error]` exists, `(boolean ...)` will return true, else false.

Reply:
This function looks much neater
```clojure
(defn retried? [job]
  (boolean (get-in job [:state :error])))